docs/application-tutorial/game_of_life.py

```diff
@@ -0,0 +1,551 @@
+import sys
+import time
+import math
+import numpy as np
+
+# Hide pygame prompt
+from os import environ
+
+environ["PYGAME_HIDE_SUPPORT_PROMPT"] = "1"
+
+import pygame
+import argparse
+
+from concrete import fhe
+
+# Constants of the game of life
```

Comment:
These constants could be near their usage. There are not really global constant.

Reply:
ok, if you want. Yes they are not constant, but I find it easier to have them all at the same place, to compare the methods

Reply:
done